### ISSUE TITLE
Addon knobs resets URL string values to defaults

### ISSUE THREAD

**Issue by binomialstew** [storybookjs/storybook#3736]
### Bug or support request summary

Navigating to a URL that includes knob values, they are reset to default. This issue was introduced between 4.0.0-alpha.6 and 7.

### Steps to reproduce

For example, run storybook examples with `yarn start`, navigate to this URL, where age should be set as 15:
http://localhost:9011/?knob-Age=15&selectedKind=Addons%7CKnobs.withKnobs&selectedStory=tweaks%20static%20values&addonPanel=storybooks%2Fstorybook-addon-knobs

You will see the age set to the default, 70.

### Please specify which version of Storybook and optionally any affected addons that you're running
```
"@storybook/addon-actions": "^4.0.0-alpha.8"
"@storybook/addon-knobs": "^4.0.0-alpha.8"
"@storybook/addon-options": "^4.0.0-alpha.8"
"@storybook/core": "^4.0.0-alpha.8"
"@storybook/react": "^4.0.0-alpha.8"
```

**Comment by binomialstew** [comment]
This shouldn't be closed

**Comment by kondei** [comment]
I want this bug to be fixed 🙇

**Comment by ndelangen** [comment]
Should be fixed in 5.1
